docs/standard/glossary.md

```diff
@@ -86,7 +86,7 @@ Garbage collector.
 
 The garbage collector is an implementation of automatic memory management.  The GC frees memory occupied by objects that are no longer in use. 
 
-See [Garbage Collection](garbage-collection.md).
+See [Garbage Collection](garbage-collection/index.md).
```

Comment:
We never (unfortunately) discussed this in-depth, but I've recently been fully qualifying these links .....

> See \[Garbage Collection](~/docs/standard/garbage-collection/index.md).

... because it at least hardens these links against **this** document moving.